.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prettier.semi": false,
+  "prettier.singleQuote": false,
+}
```

Comment:
Sure, I added since most of the styles of the repo were not following any particular style and wanted to not format on a different style on each save.

Reply:
Yeah, I think we can add prettier to the repo. I'll do that 👍